pkg/client/client.go

```diff
@@ -922,6 +968,49 @@ func (c *Client) buildProviderMigrator(migrations map[string][]byte, providerNam
 	return m, providerConfig, err
 }
 
+func createCoreSchema(ctx context.Context, pool *pgxpool.Pool) error {
```

Comment:
move this function down, usually we want to put private functions after public functions for the reader to first only see the exported interface.